### PR TITLE
Fixed the double borders on inputs; issue #6310

### LINKED ISSUE #6310: Showing double border
### Description

When I tried to search profiles and did focus on input, it was showing both blue and red borders. I am curious if this is an intentional design choice or if it may be an error. If it is a bug, I will be more than happy to work on this. 

### Screenshots

linkfreerepo

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Vee339**
**Close #6310** 

## Fixes Issue

The issue was that inputs and textarea were showing double borders when they were focused. 

## Changes proposed

The first issue was that tailwind classes were added both in the components and pages where these components were called. Therefore, I commented the classes in individual pages and they have remained now only in components that are reusable.

Secondly, there were default rings added due to tailwind CSS which I removed now.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6524"></a>

### CODE REVIEW

File: pages/search.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ export default function Search({ data }) {
         >
           <Input
             placeholder="Search user by name or tags; eg: open source,reactjs"
-            className="border-2 hover:border-tertiary-medium transition-all dark:bg-primary-high duration-250 ease-linear rounded px-6 py-2 mb-4 block w-full"
+            // className="border-2 hover:border-tertiary-medium transition-all dark:bg-primary-high duration-250 ease-linear rounded px-6 py-2 mb-4 block w-full"
```
Comment (eddiejaoude) [inline pages/search.js:157]: If commented out, I think it can be removed?

File: pages/playground.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ export default function Playground() {
         <Input
           name={gitUsername}
           value={gitUsername}
-          className="dark:bg-primary-high dark:text-white rounded border-2 hover:border-tertiary-medium"
+          // className="dark:bg-primary-high dark:text-white rounded border-2 hover:border-tertiary-medium"
```
Comment (eddiejaoude) [inline pages/playground.js:88]: If commented out, I think it can be removed?

**Comment by Vee339** [comment]
I have made the commit. Please review it.

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
It is looking better 👍 

Unfortunately for dark mode the text in the input box is not readable

**LIGHT MODE**

Screenshot 2023-05-02 at 00 10 53

**DARK MODE**

Screenshot 2023-05-02 at 00 10 58

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍

**Pull request merged**